tests/test_endpoint.py

```diff
@@ -14,12 +14,12 @@ def setUp(self):
         self.mock_ok_elasticsearch()
         self.expected_data = self.get_fixture('ok.json')
 
-    def assert_content(self, content, status='ok', **overrides):
-        data = json.loads(content)
+    def assert_response_content(self, response, status='ok', **overrides):
+        encoding = getattr(response, 'charset', 'utf-8')
```

Comment:
I checked the [JSON Specification](https://tools.ietf.org/html/rfc4627#section-3) before I wrote this...

> JSON text SHALL be encoded in Unicode.  The default encoding is UTF-8.

...but really that line of code is about supporting different versions of Django not different encodings.  Older versions of Django will not have the `charset` attribute on the response object.  In that case, we use the default value.

----

You are correct; I could have done something like this:
```python
def assert_content(self, content, status='ok', **overrides):
    data = json.loads(content.decode('utf-8'))
    # ... snip ...
```

I could certainly do that.  What do you think?